AirCasting/SessionViews/StandaloneSessionCardView.swift

```diff
@@ -0,0 +1,111 @@
+//                .background(
+//                    EmptyView()
+//                        .fullScreenCover(isPresented: $startSyncing) {
+//                            SDSyncRootView(sessionSynchronizer: sessionSynchronizer, sdSyncController: sdSyncController)
+//                        })
+        }
+    }
+    
+    var standaloneSessionCard: some View {
+        VStack(alignment: .leading, spacing: 5) {
+            header
+            content
+        }
+        .font(Fonts.regularHeading4)
+        .foregroundColor(.aircastingGray)
+        .padding()
+        .background(Color.white
+                        .shadow(color: .sessionCardShadow, radius: 9, x: 0, y: 1))
+        .overlay(Rectangle().frame(width: nil, height: 4, alignment: .top).foregroundColor(Color.red), alignment: .top)
+    }
+    
+    var header: some View {
+        SessionHeaderView(
+            action: {},
+            isExpandButtonNeeded: false,
+            isCollapsed: .constant(false),
+            session: session,
+            sessionStopperFactory: sessionStopperFactory
+        )
+    }
+    
+    var content: some View {
+        VStack(spacing: 15) {
+            Text(Strings.StandaloneSessionCardView.heading)
+                .font(Fonts.boldHeading3)
+                .foregroundColor(.darkBlue)
+            Text(Strings.StandaloneSessionCardView.description)
+                .multilineTextAlignment(.center)
+            finishAndSyncButton
+                .alert(isPresented: $showingFinishAndSyncAlert) {
+                    finishAndSyncAlert(sessionName: session.name)
+                }
+            finishAndDontSyncButton
+                .alert(isPresented: $showingFinishAlert) {
+                    SessionViews.finishSessionAlert(sessionStopper: sessionStopperFactory.getSessionStopper(for: session), sessionName: session.name)
+                }
+            .padding()
+        }
+        .padding()
+    }
+    
+    var finishAndSyncButton: some View {
+        Button(Strings.StandaloneSessionCardView.finishAndSyncButtonLabel) {
+            showingFinishAndSyncAlert = true
+        }
+        .buttonStyle(BlueButtonStyle())
+    }
+    
+    var finishAndDontSyncButton: some View {
+        Button(Strings.StandaloneSessionCardView.finishAndDontSyncButtonLabel) {
+            showingFinishAlert = true
+        }
+        .foregroundColor(.accentColor)
+    }
+    
+    func finishAndSyncAlert(sessionName: String?) -> Alert {
+        Alert(title: Text(Strings.SessionHeaderView.finishAlertTitle) +
+              Text(sessionName ?? Strings.SessionHeaderView.finishAlertTitle_2)
+              +
+              Text(" and sync from SD card?"),
+              message: Text(Strings.SessionHeaderView.finishAlertMessage_1) +
+              Text(Strings.SessionHeaderView.finishAlertMessage_2) +
+              Text(Strings.SessionHeaderView.finishAlertMessage_3) +
+              Text("\nSD card will be cleared afterwards"),
```

Comment:
Strings please

Reply:
This will be done in the sync views task